corehq/apps/app_manager/models.py

```diff
@@ -1144,7 +1144,8 @@ def get_questions(self, langs, include_triggers=False,
                 include_translations=include_translations,
             )
         except XFormException as e:
-            raise XFormException("Error in form {}".format(self.full_path_name), e)
+            raise XFormException(_('Error in form "{}": {}')
+                                 .format(trans(self.name), six.text_type(e)))
```

Comment:
In translation strings it's a good practice to use named parameters so they can be rearranged based on what makes most sense for translated text
```py
raise XFormException(_('Error in form "{name}": {error}').format(
    name=trans(self.name),
    error=six.text_type(e)),
)
```